android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java

```diff
@@ -660,6 +679,21 @@ private void addToAddressMap(String key, String stepName, String type, String[]
         return address;
     }
 
+    private String[] getDynamicRulesEngineAddress(String curKey, JSONObject curRelevance, View view, String type) {
```

Comment:
A lot of what is in this method is also in `getRulesEngineAddress`.

Seems like what is different is `RuleConstant.DYNAMIC` and `RuleConstant.RULES_FILE` could this possible be refactored into a private method that both methods can call and just pass this `RuleConstant`s as params?